app/src/main/java/vn/mbm/phimp/me/editor/editimage/EditImageActivity.java

```diff
@@ -189,7 +197,9 @@ public void onClick(View v) {
         mRotateFragment = RotateFragment.newInstance();
         mAddTextFragment = AddTextFragment.newInstance();
         mPaintFragment = PaintFragment.newInstance();
-
+        banner.setBackgroundColor(themeHelper.getPrimaryColor());
```

Comment:
ok, I will shift this in `onResume`. But I think that case is not possible with `EditImageActivity` for now, settings changes required activity to get destroyed.
But for future we can add it to onResume.